tests/unit/parsers/test.manifestjson.js

```diff
@@ -5478,4 +5478,40 @@ describe('ManifestJSONParser', () => {
       ]);
     });
   });
+
+  describe('admin_install_only', () => {
+    it.each([
+      { applications: { gecko: { admin_install_only: true } } },
+      { applications: { gecko: { admin_install_only: false } } },
+      { browser_specific_settings: { gecko: { admin_install_only: true } } },
+      { browser_specific_settings: { gecko: { admin_install_only: false } } },
+    ])(
+      'emits an error when the admin_install_only flag is present - %o',
+      (manifestProps) => {
+        const linter = new Linter({ _: ['bar'] });
+
+        const manifestJSONParser = new ManifestJSONParser(
+          JSON.stringify({
+            manifest_version: 2,
```

Comment:
Thought: similarly I wonder if here we should test it out of both mv2 and mv3 extensions, just to make sure that if we tweak the implementation we can't mistakenly end up only checking this property for a particular manifest version and not for the other one.